pyscf/pbc/dft/numint.py

```
@@ -1159,8 +1160,9 @@ def nr_rks(self, cell, grids, xc_code, dms, relativity=0, hermi=1,
                kpts=None, kpts_band=None, max_memory=2000, verbose=None, **kwargs):
         if kpts is None:
             if 'kpt' in kwargs:
-                sys.stderr.write('WARN: KNumInt.nr_rks function finds keyword '
-                                 'argument "kpt" and converts it to "kpts"\n')
+                logger.warn(self, 'KNumInt.nr_rks function finds '
```

Please raise an error for this incorrect kwarg.